composer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "wordpress/twentynineteen",
+  "description": "A new Gutenberg-ready theme.",
+  "type": "package",
+  "license": "GPL v2 or later",
+  "keywords": [
+		"WordPress",
+		"Themes"
+	],
+  "authors": [
+		{
+			"name": "Contributors",
```

Comment:
The authors name should also be taken from the `style.css` file, `the WordPress team`

https://github.com/WordPress/twentynineteen/blob/master/style.css#L5